pkg/controllers/vscr/vdbverify_reconciler.go

```diff
@@ -93,6 +94,16 @@ func (s *VDBVerifyReconciler) checkVersionAndDeploymentType(ctx context.Context)
 			"The server version %s does not have scrutinize support through vclusterOps", ver)
 		return s.updateScrutinizeReadyCondition(ctx, metav1.ConditionFalse, events.VclusterOpsScrutinizeNotSupported)
 	}
+
+	if vinf.IsOlder(v1.ScrutinizeDBPasswdInSecretMinVersion) {
+		ver, _ := s.Vdb.GetVerticaVersionStr()
+		s.VRec.Eventf(s.Vscr, corev1.EventTypeWarning, events.VclusterOpsScrutinizePasswdInSecretNotSupported,
+			"The server version %s does not support vclusterOps scrutinize reading db password from secret", ver)
```

Comment:
Can we shorten this to say it only runs with 24.2.0?
```suggestion
			"The server version %s is not supported with VerticaScrutinize. The minimum server version it supports is %s.", ver, v1.ScrutinizeDBPasswdInSecretMinVersion)
```